examples/c/example.c

```diff
@@ -569,11 +569,11 @@ AranyaError init_team(Team* t) {
     // assign role management permissions.
     err = aranya_assign_role_management_permission(
         &owner->client, &t->id, &operator_role_id, &admin_role_id,
-        "CanAssignRole");
+        ARANYA_ROLE_MANAGEMENT_PERMISSION_CAN_ASSIGN_ROLE);
```

Comment:
This is really long. Could be shortened to:
```suggestion
        ARANYA_PERM_CAN_ASSIGN_ROLE);
```
We would just need to make sure regular perm names don't overlap with management perm names.